composer/algorithms/cutmix/cutmix.py

```diff
@@ -200,13 +206,19 @@ class CutMix(Algorithm):
             )
     """
 
-    def __init__(self, num_classes: int, alpha: float = 1., uniform_sampling: bool = False):
+    def __init__(self,
+                 num_classes: int,
+                 alpha: float = 1.,
+                 uniform_sampling: bool = False,
+                 input_key: Union[Any, Callable, Sequence[Callable[[], Any]]] = 0,
```

Comment:
Hmm, I think this Union annotation makes it more confusing than just typing it as `Any`. Mentioning only the callable case might confuse one that they have to pass in a callable (which hides all the magic of your batch_get / batch_set functions!)

If we did want to give a hint, would `Union[int, str, Callable, Any]` work? But I think I'd still prefer just `Any` with an explanation in the docstring...